pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -853,3 +854,14 @@ func (r *ReconcileJenkinsBaseConfiguration) ensureBaseConfiguration(jenkinsClien
 
 	return reconcile.Result{Requeue: requeue}, err
 }
+
+func (r *ReconcileJenkinsBaseConfiguration) isOpenshiftEnabled() bool {
+	openshiftEnabled := false
+	if r.Configuration.Jenkins.Annotations[openshiftEnabledAnnotation] == "true" {
+		openshiftEnabled = true
+		r.logger.Info("-Openshift Mode-")
```

Comment:
I don't think that printing to the log is necessary. Also every time you call `isOpenshiftEnabled` function this line will print the message every time that function is being called.

Reply:
makes sense I'll do that

Reply:
come to think of it I will change the var to `isOpenshift` and keep `openshift mode` as openshift is not something that is enabled if that makes sense